fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreIndexTest.java

```diff
@@ -2556,7 +2557,7 @@ public void formerIndexes() throws Exception {
             uncheckedOpenRecordStore(context, metaData.getRecordMetaData());
             timer.reset();
             recordStore.checkVersion(null, FDBRecordStoreBase.StoreExistenceCheck.ERROR_IF_NOT_EXISTS).join();
-            assertEquals(1, timer.getCount(FDBStoreTimer.Events.REBUILD_INDEX));
+            assertEquals(1, timer.getCount(FDBStoreTimer.Events.REBUILD_INDEX_FEW_RECORDS));
```

Comment:
This now emits both kpis, it seems like we should restore the assertions about `REBUILD_INDEX` to these tests?

Reply:
✅

Reply:
Sorry, I wasn't clear, I meant that it should assert both about `REBUILD_INDEX` and the new events that you added.

Reply:
That makes more sense :)